src/vos/tests/evt_ctl.c

```diff
@@ -1212,13 +1223,22 @@ test_evt_iter_delete(void **state)
 
 		sum += *value;
 		utest_free(arg->ta_utx, addr.ba_off);
+		if (um_inst->umm_id != UMEM_CLASS_VMEM) {
+			rc = utest_check_mem_decrease(arg->ta_utx);
+			assert_int_equal(rc, 0);
+			rc = utest_sync_mem_status(arg->ta_utx);
+			assert_int_equal(rc, 0);
+		}
 	}
 	rc = evt_iter_finish(ih);
 	assert_int_equal(rc, 0);
 
 	expected_sum = NUM_EPOCHS * (NUM_EXTENTS * (NUM_EXTENTS + 1) / 2);
 	assert_int_equal(expected_sum, sum);
-
+	if (um_inst->umm_id != UMEM_CLASS_VMEM) {
```

Comment:
you can remove these checks now, no?

Reply:
Moved all the checks under utest_common.c and removed from evt_ctl.c